app.js

```diff
@@ -207,30 +207,41 @@ function orchestrate(spec, isManifest, token) {
                                         spec.jobs['publish'].status = 'ok';
 
                                         spec.jobs['tr-install'].status = 'pending';
-                                        finalTRpath = report.metadata.get('thisVersion').replace(W3C_PREFIX, '');
-                                        return trInstaller(tempLocation, finalTRpath).then(function () {
-                                            spec.jobs['tr-install'].status = 'ok';
-
-                                            spec.jobs['update-tr-shortlink'].status = 'pending';
-                                            return updateTrShortlink(report.metadata.get('thisVersion')).then(function () {
-                                                spec.jobs['update-tr-shortlink'].status = 'ok';
-
-                                                var cmd = global.SENDMAIL + ' ' + global.MAILING_LIST + ' ' + report.metadata.get('thisVersion');
-                                                exec(cmd, function (err, stdout, stderr) {
-                                                  if (err) console.error(stderr);
+                                        if (0 === report.metadata.get('thisVersion').indexOf(W3C_PREFIX)) {
```

Comment:
I'm not sure to understand what is the purpose of this `if` statement. If the document makes it that far, then the *This Version* URL passes the checkers, and is therefore a valid `http://www.w3.org/TR/[...]`.
Am I being mistaken here?